lib/r10k/settings/definition.rb

```diff
@@ -90,7 +90,7 @@ def validate
       def resolve
         if !@value.nil?
           @value
-        elsif @default
+        elsif !@default.nil?
```

Comment:
Why is this needed?

Reply:
The original logic was faulty because if the `@default` was `false`, then this conditional—`if false`—wouldn't work and you would end up with a default of `nil`, which is ostensibly `false` but not the actual default.

Reply:
ahh, good point!